src/components/templates/Auditorium/Auditorium.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from "react";
+
+import { SectionPreviews } from "./components/SectionPreviews";
+import { Section } from "./components/Section";
+
+import { AuditoriumVenue } from "types/venues";
+
+const enum AuditoriumVideoPosition {
+  TOP = "top",
+  CENTER = "center",
+}
```

Comment:
This whole PR in a draft mode. The location of the vars will change/types/components. The reason I opened it is to get the `high-level` opinion on the way it is implemented, not the small things

Reply:
I'm aware that this is draft mode, and while yes, I am absolutely going to give the high level of things, i'm also going to point out aspects (without going too 'deep' into the nitty gritty) that I see as being important. This is one of those things. Whether you choose to do that now or later is irrelevant to me, but i'm not going to not comment on things like this just because. It takes me more time and effort to not, and I would much rather it be said from the beginning so that the patterns can remain enforced throughout.

If you don't want me to do this in future, and you're already aware/planning to do a thing, feel free to have a `// TODO:` comment describing the intent, or add your own review comments calling out the things you intend to do with it, and I will happily skip past them knowing that they are 'known and accounted for'.

In this instance, assuming you already knew/were planning to do it, a 'thumbs up' would have been sufficient.